Source/WPEFramework/PluginServer.h

```diff
@@ -1725,12 +1725,13 @@ PUSH_WARNING(DISABLE_WARNING_THIS_IN_MEMBER_INITIALIZER_LIST)
                     config.VolatilePath(), 
                     config.AppPath(), 
                     config.ProxyStubPath(), 
-                    config.PostMortemPath(), 
+                    config.PostMortemPath(),
                     _engine)
                 , _server(server)
                 , _subSystems(this)
                 , _authenticationHandler(nullptr)
             {
+                _processAdministrator.ForcedDestructionTimes(config.SoftKillCheckWaitTime(), config.HardKillCheckWaitTime());
```

Comment:
Why not pass them on via the CommunicatorServer constructor?  And via that to the Communicator constructor

Reply:
@pwielders, suggested to create method called ForcedDestructionTimes in Communicator class to set these values. Hence I used the method here